src/types/commands.ts

```diff
@@ -1032,13 +1053,12 @@ export type CoreCommand =
   | CreateChartCommand
   | UpdateChartCommand
 
-  /** Image */
+  /** IMAGE */
```

Comment:
Useless diff, or we should change every comments here, as we have "FILTER" but "Grouping"

Reply:
I might as well fix everything.